src/cms/widgets/LocalizedString/stories.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+import '../../../scss/main.scss'
+import { storiesOf } from '@storybook/react'
+import cn from 'classnames'
+import styles from './styles.module.scss'
+
```

Comment:
Either remove to stay consistent with the current code base or check the [sorting rules](https://eslint.org/docs/rules/sort-imports) and maybe we find a better solution for the imports.

```suggestion
```